cpp/src/io/orc/writer_impl.cu

```diff
@@ -1018,6 +1018,7 @@ std::vector<StripeInformation> writer::impl::gather_stripes(
   hostdevice_2dvector<gpu::encoder_chunk_streams>* enc_streams,
   hostdevice_2dvector<gpu::StripeStream>* strm_desc)
 {
+  if (segmentation.num_stripes() == 0) return {};
```

Comment:
In the interest of satisfying a near-future change from clang-tidy (#9860):
```suggestion
  if (segmentation.num_stripes() == 0) { return {}; }
```

Reply:
I tried to suggest that style many times, and so happy now that such style will be enforced :joy:.

Reply:
I use it almost everywhere, just seems fairly pointless for branches that return.